examples/seismic/utils.py

```diff
@@ -193,6 +193,23 @@ def __call__(self, parser, args, values, option_string=None):
             values = {'float32': np.float32, 'float64': np.float64}[values]
             setattr(args, self.dest, values)
 
+    class _opt_action(Action):
```

Comment:
that's duplicate from benchmark, can we put it somewhere common?

Reply:
they use differ arg parsers so I'm not sure?

Reply:
Hum slightly different reutrn type indeed, I guess it's fine